content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md

```diff
@@ -108,7 +108,7 @@ this example.
     If you already have a CA then the only action that is copying the CA's `crt` and
     `key` file to `/etc/kubernetes/pki/etcd/ca.crt` and
     `/etc/kubernetes/pki/etcd/ca.key`. After those files have been copied, please
-    skip this step.
+    proceed to "Create certificates for each member".
```

Comment:
Remove "please" and link to the step you should go to, rather than using quotes.